### ISSUE TITLE
Everything is reading loading (Bullseye Pi4) 

### ISSUE THREAD

**Issue by lundyfpv** [RotorHazard/RotorHazard#637]
**Describe the bug**
Fresh install of 3.1.0 followed the guide, had to change some of the python packages as noted in a seperate issue. Server starts up just fine and everything appears to be working. Get into settings to setup Freqs and, Events, Pilots, etc and everything starts reporting as loading after everything gets setup. 

**To Reproduce**
I did a clean install and using an S32_BPill PCB that was working on a previous version. 

**Expected behavior**
For it not to always say loading. 

**Screenshots**
There are Heats and Pilots setup here and I can make them show if I click "Add Heat" or "Add Pilot".
![image](https://user-images.githubusercontent.com/8151341/144668674-8faa4e5a-4dcb-4673-9e83-a3544f7e779e.png)

After clicking: 
![image](https://user-images.githubusercontent.com/8151341/144668844-b9dc07e2-a6a8-4ca6-ad2f-f9b23d81aa6b.png)

Main Events Tab
![image](https://user-images.githubusercontent.com/8151341/144669282-d8710a0a-3e79-45eb-b4f2-31209a7ebb53.png)

I setup Nodes 1-4 (RB1,RB2,RB7,RB8) but this is all I see:
![image](https://user-images.githubusercontent.com/8151341/144669695-37bf42bf-a264-488f-a2a2-e7e90a9433c2.png)

**Log Dump**
[rh_logs_20211203_202055.zip](https://github.com/RotorHazard/RotorHazard/files/7652199/rh_logs_20211203_202055.zip)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Chrome/Edge
 - Version 96+

**Comment by ethomas997** [comment]
Thanks for the report. Looks like a bug; we'll fix it soon.  As a temporary workaround, if you can remove the top two RX5808 modules and run with 4 nodes it may get it working.

**Comment by lundyfpv** [comment]
That does seem to make everything work as expected. Thanks for the quick response @ethomas997!

**Comment by ethomas997** [comment]
Looks like this happens when nodes are added to an existing setup.  We'll be implementing a fix, but another workaround is to delete the "src/server/database.db" file (after stopping the RotorHazard server if running).  When it restarts it should create a new database.db file that runs correctly.

**Comment by lundyfpv** [comment]
The nodes were attached on first load of the server so they weren't added afterwards if that's what you mean.

**Comment by ethomas997** [comment]
In your log files it looks like there was an initial startup with no nodes attached, which had a similar effect.  In any case, the server code needs a fix to make it handle this type of thing gracefully.

**Comment by lundyfpv** [comment]
Ah yes that makes sense I did all the setup to the Pu before connecting the main pcb.

**Comment by ethomas997** [comment]
I've posted a fix; to update to the fixed version (3.1.1-dev.1) use these commands:
```
cd ~
wget https://codeload.github.com/RotorHazard/RotorHazard/zip/3.1.1-dev.1 -O temp.zip
unzip temp.zip
mv RotorHazard RotorHazard.old
mv RotorHazard-3.1.1-dev.1 RotorHazard
rm temp.zip
cp RotorHazard.old/src/server/config.json RotorHazard/src/server/
cp RotorHazard.old/src/server/database.db RotorHazard/src/server/
```
With this version you should be able to add nodes without any trouble.
